ImageLounge/src/DkGui/DkNoMacs.cpp

```diff
@@ -2131,13 +2155,18 @@ void DkNoMacs::setWindowTitle(QSharedPointer<DkImageContainerT> imgC) {
 
 void DkNoMacs::setWindowTitle(const QString& filePath, const QSize& size, bool edited, const QString& attr) {
 
+	QString title;
+
 	// TODO: rename!
+	if (getTabWidget()->getTabs().count() > 1) {

```

Comment:
I do understand that this is needed for `Go To` tab but I don't like to have much information in the title bar (there were lot's of requests to put other information there too). Moreover, there are a few update glitches (e.g. wen opening a new tab) - so please remove it.